core/app/models/spree/promotion/rules/first_purchase_since.rb

```diff
@@ -0,0 +1,36 @@
+module Spree
+  class Promotion
+    module Rules
+      class FirstPurchaseSince < PromotionRule
+        preference :months_ago, :integer, default: 12
+        validates :preferred_months_ago, numericality: { only_integer: true, greater_than: 1 }
```

Comment:
No technical reason this couldn't be 1, right? So should this be `greater_than_or_equal_to`?